### PR TITLE
wd_demo.au3 - ability to repeat "UserFile" processing

### PR DESCRIPTION
**Pull request by mlipok**
### Proposed changes

As I mention before: https://github.com/Danp2/au3WebDriver/pull/462#issuecomment-1522290146

### Checklist

- [x] I have read and noticed the [CODE OF CONDUCT](https://github.com/Danp2/au3WebDriver/blob/master/docs/CODE_OF_CONDUCT.md) document
- [x] I have read and noticed the [CONTRIBUTING](https://github.com/Danp2/au3WebDriver/blob/master/docs/CONTRIBUTING.md) document
- [ ] I have added necessary documentation or screenshots (if appropriate)

### Types of changes
- [ ] Bugfix (change which fixes an issue)
- [x] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (functional, structural)
- [ ] Documentation content changes
- [ ] Other (please describe)

### What is the current behavior?

`UserFile()` process `UserTesting.au3` only once and after processing close browser

### What is the new behavior?

You can choose new or the same `UserTesting.au3` and process them again without closing browser.

### Additional context

Was tested with `usertesting.au3` modified with this PR, and with separate `usertesting_WD_GetContext.au3` which contains:
```autoit
ConsoleWrite("! START" & @CRLF)

__SetVAR(0, "Absolute Identifiers > _WD_FrameEnter|Relative Identifiers > _WD_FrameEnter|IFRAME/FRAME attributes|URL|Body ElementID|IsHidden|MatchedElements")
ConsoleWrite("! 0 - $_VAR[0] = " & $_VAR[0] & @CRLF)

_Demo_NavigateCheckBanner($sSession, "https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_iframe", '//*[@id="snigel-cmp-framework" and @class="snigel-cmp-framework"]')
ConsoleWrite("! 1" & @CRLF)

__SetVAR(1, _WD_GetContext($sSession))
ConsoleWrite("! 2 _WD_GetContext >> " & $_VAR[1] & @CRLF)

__SetVAR(2, _WD_FrameList($sSession))
ConsoleWrite("! 3" & @CRLF)

_ArrayDisplay($_VAR[2], 'Starting', 0, 0, Default, $_VAR[0])
ConsoleWrite("! 4 - _ArrayDisplay" & @CRLF)

_WD_FrameEnter($sSession, 'null/0')
ConsoleWrite("! 5" & @CRLF)

__SetVAR(3, _WD_FrameList($sSession))
ConsoleWrite("! 6" & @CRLF)

_ArrayDisplay($_VAR[3], 'null/0', 0, 0, Default, $_VAR[0])
ConsoleWrite("! 7 - _ArrayDisplay" & @CRLF)

_WD_FrameEnter($sSession, $_VAR[1])
ConsoleWrite("! 8 after entering frame taken from _WD_GetContext()" & @CRLF)

__SetVAR(4, _WD_FrameList($sSession))
ConsoleWrite("! 9" & @CRLF)

_ArrayDisplay($_VAR[4], 'relative CONTEXT = ' & $_VAR[1], 0, 0, Default, $_VAR[0])
ConsoleWrite("! 10 - _ArrayDisplay" & @CRLF)
```

### System under test
not related

### CODE REVIEW

File: usertesting.au3 [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
 _WD_LoadWait($sSession, 1000)
-_WD_FindElement($sSession, $_WD_LOCATOR_ByXPath, "//input[@title='Search']")
 
 ConsoleWrite("- Test 2:" & @CRLF)
-_WD_WaitElement($sSession, $_WD_LOCATOR_ByCSSSelector, '#fake', 1000, 3000, $_WD_OPTION_NoMatch)
\ No newline at end of file
+_WD_NewTab($sSession, True)
+_WD_Navigate($sSession, 'https://www.google.com')
+_WD_LoadWait($sSession)
+MsgBox($MB_OK + $MB_TOPMOST + $MB_ICONWARNING, "Warrning #" & @ScriptLineNumber, "If you see COOKIE accept panel close them before you will continue.")
```
Comment (Danp2) [inline usertesting.au3:16]: typo

File: wd_demo.au3 [hunk 2]
```diff
@@ -1139,27 +1139,34 @@ EndFunc   ;==>UserTesting
...
+		If @error Then Return SetError(@error, @extended)
+
+		For $sCmd In $aCmds
+			; Strip comments
+			; https://www.autoitscript.com/forum/topic/157255-regular-expression-challenge-for-stripping-single-comments/?do=findComment&comment=1138896
+			$sCmd = StringRegExpReplace($sCmd, '(?m)^(?:[^;"'']|''[^'']*''|"[^"]*")*\K;.*', "")
+			If $sCmd Then Execute($sCmd)
+		Next
+
+	Until $IDNO = MsgBox($MB_YESNO + $MB_TOPMOST + $MB_ICONQUESTION + $MB_DEFBUTTON2, 'Question', 'Do you want to repeat "UserFile" processing ?')
```
Comment (Danp2) [inline wd_demo.au3:1162]: Perhaps change the question to "Do you want to process another file?"

File: usertesting.au3 (hunk 1 above)
Comment (mlipok) [reply usertesting.au3]: Fixed: [typo](https://github.com/Danp2/au3WebDriver/pull/464/commits/4fa519d5ab3db59851610911746cd201442f3f5f)

File: wd_demo.au3 (hunk 2 above)
Comment (mlipok) [reply wd_demo.au3]: As you suggested [question changed](https://github.com/Danp2/au3WebDriver/pull/464/commits/86f9d4e4dcd6e9e95f8cd7839dd4184244727edb)

**Pull request merged**